### PR TITLE
Added example how to use ollama hosted models

### PR DESCRIPTION
**Pull request by grg-ffb**
Example added how to use llama3.2 locally run via Ollama

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -459,6 +459,36 @@ answer = ask(
 )
 ```
 
+Also there is the possibility to use ollama hosted models.
```
Comment (mskarlin) [inline README.md:462]:
```suggestion
Models hosted with `ollama` are also supported.
```

File: README.md [hunk 2]
```diff
@@ -459,6 +459,36 @@ answer = ask(
...
+Tun run example below make sure you have downloaded llama3.2 and mxbai-embed-large via ollama.
```
Comment (mskarlin) [inline README.md:463]:
```suggestion
To run the example below make sure you have downloaded llama3.2 and mxbai-embed-large via ollama.
```

**Review by mskarlin — COMMENTED** [review]
Thanks for your contribution, I made a suggestion for a minor edit!

**Comment by jamesbraza** [comment]
Hi @grg-ffb this looks great, if you can get `pre-commit` to pass we can merge this in.

**Comment by jamesbraza** [comment]
Thank you @grg-ffb ! Appreciated

I am not sure how much time you have, but having a unit test that echoes these docs on Ollama + PaperQA in `tests/test_paperqa.py` could be cool

**Pull request merged**